UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/AzureWebJobHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Newtonsoft.Json;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    [ExcludeFromCodeCoverage]
+    public class AzureWebJobHelper : IAzureWebJobHelper
+    {
+        static HttpClient httpClient = new HttpClient();
```

Comment:
new assignment can be simplified and it can be made readonly as:
static readonly HttpClient httpClient = new();

Reply:
Changes done